Added firewall update command

stuclem opened the pull request:
Per title. 

Fixes https://github.com/vmware/vic-product/issues/45.

@andrewtchin can you please take a look? Thanks!

andrewtchin commented on docs/user_doc/vic_vsphere_admin/ts_firewall_error.md:
```diff
@@ -12,37 +12,24 @@ ESXi hosts communicate with the VCHs through port 2377 via Serial Over LAN. For
-</pre>
+The `vic-machine` utility includes an `update firewall` command, that you can use to modify the firewall on the ESXi host or the ESXi hosts in a cluster. 
+
+You use `--allow` and `--deny` flags to enable and disable the `vSPC` ruleset. When enabled, the `vSPC` rule allows all outbound TCP traffic from the target host or hosts. If you disable the rule, you must configure the firewall via another method to allow outbound connections on port 2377 over TCP. If you do not enable the rule or configure the firewall, vSphere Integrated Containers does not fu ...
```
VCHes?

stuclem replied:
Nope! Since VCH is already an abbreviation, there's no need to add the 'e'. Even if the plural of "sandwich" is "sandwiches" :)

andrewtchin commented on docs/user_doc/vic_vsphere_admin/ts_firewall_error.md:
````diff
@@ -12,37 +12,24 @@ ESXi hosts communicate with the VCHs through port 2377 via Serial Over LAN. For
 
-  ```$ esxcli network firewall set --enabled false``` 
\ No newline at end of file
+**NOTE**: Setting the `vSPC` rule does not persist through an ESXi host reboot. If you reboot an ESXi host on which you ran `vic-machine-windows update firewall` to enable the rule, you must run `vic-machine-windows update firewall` again to reenable the rule.
````
On both my fusion ESXi 6.5 and @chengwang86's latest build, when using the `update firewall` command the rule persisted between restarts. I would also think persistence would be the expected behavior because this command is using the same API as the UI and I wouldn't expect the user to have to reconfigure those every reboot. Could you recheck this?

stuclem replied:
I had made a note in the issue that stated that the rule isn't persistent, but I just rebooted my ESX, saw that the `vSPC` rule was still set to `true`, and was able to deploy a VCH. So, I removed the note about it not persisting.

stuclem commented:
@andrewtchin I thought about this a bit and decided that this info needs to be in a topic of its own, right up front before you start deploying VCHs, rather than in a troubleshooting topic that tells you what you should have done before you tried to deploy a VCH and hit the firewall error. So I added the info in a new topic, and updated the troubleshooting topic to point to the new topic. All existing links to the TS topic now point to the new one. I think that this is better.

Can you please take another look?

andrewtchin commented:
lgtm

stuclem commented:
Thanks @andrewtchin !
